examples/parallel_csv_ingestion.py

```diff
@@ -274,7 +274,8 @@ def test_parallel_csv_ingestion():
         "column_int64": np.int64,
         "column_uint32": np.uint32,
         "column_float64": np.float64,
-        "column_utf8": np.str,
+        # Avoid this runtime warning: "DeprecationWarning: `np.str` is a deprecated alias for the builtin `str`."
```

Comment:
nit: inline comment probably not necessary

Reply:
derp, thanks! i'll get this on another PR ... :)